### DISCUSSION TITLE
Copilot Question

### DISCUSSION THREAD

**Discussion by JacobScheff** [community/community#24688] [Apps, API and Webhooks]
<p>I signed up for Copilot about 2 months and a half ago and still haven’t received access yet. However, my friend signed up only about one month ago and they got access already. Is it normal to have to wait longer than others to get access to Copilot?</p>

**Comment by Zerotask** [comment, marked as answer]
<p>Yes, you just have to be patient.</p>

**Comment by JacobScheff** [comment]
<p>I just got access to it.</p>
